modules/tinymce/src/plugins/link/test/ts/browser/SelectedImageFigureLinkTest.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, it, before, after } from '@ephox/bedrock-client';
+import { TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/link/Plugin';
+
+import { TestLinkUi } from '../module/TestLinkUi';
+
+describe('browser.tinymce.plugins.link.SelectedImageFigureTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'link image',
+    toolbar: 'link',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin ]);
+
+  before(() => {
+    TestLinkUi.clearHistory();
+  });
+
+  after(() => {
+    TestLinkUi.clearHistory();
+  });
+
+  it('TINY-8832: link button should NOT be highlighted when there is no link in a figure element', async () => {
```

Comment:
This description doesn't really cover what's being tested now? This is why I preferred the 2 separate tests to be honest, as now we're testing multiple things within an `it`.

- [ ] Consider reverting back to using 2 separate `it` blocks

Reply:
It was my initiative to join those two tests in order to avoid adding specific delays in tests like `Waiter.pWait(100)`

Reply:
Hmm... fair it had all changed since you originally reviewed it. If we do need a `pWait` I'm fine leaving it as a single test.

Reply:
I reverted back to using two tests since we don't need a `pWait` anymore